### PR TITLE
Garage Limit (rework)

### PR DESCRIPTION
**Pull request by DardoTheMaster**
Hi all
As the previous pull request,these edits allow server admins to set a limit for the vehicles that each player is able to store. I'd like someone to test it (again) before an eventual merge.

Hope the feature will get appreciated!

### CODE REVIEW

File: Altis_Life.Altis/Config_Master.hpp [hunk 1]
```diff
@@ -115,7 +115,7 @@ class Life_Settings {
     vehicle_infiniteRepair = false; //Set to true for unlimited repairs with 1 toolkit. False will remove toolkit upon use.
     vehicleShop_rentalOnly[] = { "B_MRAP_01_hmg_F", "B_G_Offroad_01_armed_F", "B_Boat_Armed_01_minigun_F" }; //Vehicles that can only be rented and not purchased. (Last only for the session)
     vehicleShop_3D = false; //Add preview 3D inside Shop vehicle.       Default : False
-
+    garage_limit = false;
```
Comment (setoy) [inline Altis_Life.Altis/Config_Master.hpp]: What about a comment regarding possible values?

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -526,7 +526,7 @@
             <Original>Debug</Original>
             <Czech>Ladit</Czech>
             <Spanish>Debug</Spanish>
-            <Italian>mettere a punto</Italian>
+            <Italian>Debug</Italian>
             <Russian></Russian>
```
Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: Huh?

File: Altis_Life.Altis/stringtable.xml [hunk 3]
```diff
@@ -3317,12 +3317,17 @@
             <Portuguese>Não existe um veículo por perto...</Portuguese>
             <Polish>W pobliżu nie ma pojazdu...</Polish>
         </Key>
+       <Key ID="STR_NOTF_TooMuchVehicles">
+     <Original>You can't store more than %1 vehicles</Original>
```
Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: Indenting?

**Comment by DardoTheMaster** [comment]
Broken pull

**Pull request closed without merging**